src/test/java/com/getindata/connectors/http/internal/table/lookup/querycreators/ElasticSearchLiteQueryCreatorTest.java

```diff
@@ -18,6 +18,24 @@
 
 public class ElasticSearchLiteQueryCreatorTest {
 
+    @Test
```

Comment:
I think this PR should not contain irrelevant changes, so I added these tests in PR: https://github.com/getindata/flink-http-connector/pull/130
Please rebase your branch with main :)

Reply:
@grzegorz8 no worries - I was just pulling in the fix as is from Kristof as a base.